<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>BallspeedUI</class>
    <widget class="QWidget" name="mainWidget">
    <property name="windowTitle">
        <string>Ballspeed</string>
    </property>
    <property name="geometry">
        <rect>
            <x>0</x>
            <y>0</y>
            <width>800</width>
            <height>600</height>
        </rect>
    </property>
        <layout class="QGridLayout">
            <item row="0" column="0" colspan="2">
                <layout class="QHBoxLayout" name="topLayout">
                    <item>
                        <widget class="QLineEdit" name="videoPath">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="loadVideoButton">
                            <property name="text">
                                <string>Load ...</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="1" column="0">
                <layout class="QVBoxLayout" name="settingsLayout">
                    <item>
                        <widget class="QLabel" name="ballDiametersLabel">
                            <property name="text">
                                <string>Ball diameter</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="ballDiameter">
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="goalHeightLabel">
                            <property name="text">
                                <string>Goal height</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="goalHeight">
                        </widget>
                    </item>
<!--                    <item>
                        <widget class="QLabel" name="contrastLabel">
                            <property name="text">
                                <string>Contrast</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QSlider" name="contrast">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>-->
                    <item>
                        <widget class="QLabel" name="diameterLabel">
                            <property name="text">
                                <string>Diameter</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QSlider" name="diameter">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>5</number>
                            </property>
                            <property name="maximum">
                                <number>200</number>
                            </property>
                            <property name="value">
                                <number>50</number>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="houghLabel">
                            <property name="text">
                                <string>Hough image</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="hough">
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="houghThresholdLabel">
                            <property name="text">
                                <string>Hough threshold</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QSlider" name="houghThreshold">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>5</number>
                            </property>
                            <property name="maximum">
                                <number>200</number>
                            </property>
                            <property name="value">
                                <number>50</number>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="templateLabel">
                            <property name="text">
                                <string>Template</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="templateImage">
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="calculateSpeed">
                            <property name="text">
                                <string>Calculate speed</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="matchTemplate">
                            <property name="text">
                                <string>Match template</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout">
                            <item>
                                <widget class="QPushButton" name="matchAllBalls">
                                    <property name="text">
                                        <string>Match # of frames</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="frameCount">
                                    <property name="text">
                                        <string>25</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item row="1" column="1">
                <layout class="QVBoxLayout">
                    <item>
                        <widget class="VideoFrame" name="frame">
                            <property name="minimumSize">
                                <size>
                                    <width>800</width>
                                </size>
                            </property>
                            <property name="alignment">
                                <enum>Qt::AlignTop</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout">
                            <item>
                                <widget class="QPushButton" name="firstFrame">
                                    <property name="text">
                                        <string>&lt;&lt;</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="prevFrame">
                                    <property name="text">
                                        <string>&lt;</string>
                                    </property>
                                </widget>
                            </item>
                            <item><widget class="QLineEdit" name="frameEdit" /></item>
                            <item>
                                <widget class="QPushButton" name="nextFrame">
                                    <property name="text">
                                        <string>&gt;</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="lastFrame">
                                    <property name="text">
                                        <string>&gt;&gt;</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
</ui>
                                

